addons/l10n_ie/data/template/account.fiscal.position-ie.csv

```diff
@@ -0,0 +1,45 @@
+"id","sequence","name","auto_apply","vat_required","country_id","country_group_id","tax_ids/tax_src_id","tax_ids/tax_dest_id","account_ids/account_src_id","account_ids/account_dest_id"
+"ie_fp_domestic","1","Domestic","1","1","base.ie","","","","",""
+"ie_fp_eu_private","2","EU private","1","","","base.europe","","","",""
+"ie_fp_eu","3","Intra-community (EU)","1","1","","base.europe","ie_tax_sale_good_23","ie_tax_sale_good_eu_0","",""
+"","","","","","","","ie_tax_sale_service_23","ie_tax_sale_service_eu_0","",""
+"","","","","","","","ie_tax_sale_good_13_5","ie_tax_sale_good_eu_0","",""
+"","","","","","","","ie_tax_sale_service_13_5","ie_tax_sale_service_eu_0","",""
+"","","","","","","","ie_tax_sale_good_9","ie_tax_sale_good_eu_0","",""
+"","","","","","","","ie_tax_sale_service_9","ie_tax_sale_service_eu_0","",""
+"","","","","","","","ie_tax_sale_good_0","ie_tax_sale_good_eu_0","",""
+"","","","","","","","ie_tax_sale_service_0","ie_tax_sale_service_eu_0","",""
```

Comment:
NBD: It seems you forgot to translate the `ie_tax_sale_farm_5` and `ie_tax_sale_ls_4_8` taxes into `ie_tax_sale_good_eu_0` here (like you did for the Import/Export fiscal position below)